doc/reference/configuration/configuration_reference.rst

```diff
@@ -1671,6 +1671,114 @@ See also: :ref:`supervised_failover_overview_fault_tolerance`
     | Default: 2
     | Environment variable: TT_FAILOVER_STATEBOARD_RENEW_INTERVAL
 
+..  _configuration_reference_feedback:
+
+feedback
+--------
+
+The ``feedback`` section describes configuration parameters for sending information about a running Tarantool instance to the specified feedback server.
+
+..  NOTE::
+
+    ``feedback`` can be defined in any :ref:`scope <configuration_scopes>`.
+
+* :ref:`feedback.crashinfo <configuration_reference_feedback_crashinfo>`
+* :ref:`feedback.enabled <configuration_reference_feedback_enabled>`
+* :ref:`feedback.host <configuration_reference_feedback_host>`
+* :ref:`feedback.interval <configuration_reference_feedback_interval>`
+* :ref:`feedback.metrics_collect_interval <configuration_reference_feedback_metrics_collect_interval>`
+* :ref:`feedback.metrics_limit <configuration_reference_feedback_metrics_limit>`
+* :ref:`feedback.send_metrics <configuration_reference_feedback_send_metrics>`
+
+..  _configuration_reference_feedback_crashinfo:
+
+..  confval:: feedback.crashinfo
+
+    Whether to send crash information in the case of an instance failure.
+    This information includes:
+
+    -   General information from the ``uname`` output.
+    -   Build information.
+    -   The crash reason.
+    -   The stack trace.
+
+    To turn off sending crash information, set this option to ``false``.
+
+    |
+    | Type: boolean
+    | Default: true
+    | Environment variable: TT_FEEDBACK_CRASHINFO
+
+..  _configuration_reference_feedback_enabled:
+
+..  confval:: feedback.enabled
+
+    Whether to send information about a running instance to the feedback server.
+    To turn off sending feedback, set this option to ``false``.
+
+    |
+    | Type: boolean
+    | Default: true
+    | Environment variable: TT_FEEDBACK_ENABLED
+
+..  _configuration_reference_feedback_host:
+
+..  confval:: feedback.host
+
+    The address to which information is sent.
+
+    |
+    | Type: string
+    | Default: https://feedback.tarantool.io
+    | Environment variable: TT_FEEDBACK_HOST
+
+..  _configuration_reference_feedback_interval:
+
+..  confval:: feedback.interval
+
+    The interval (in seconds) of sending information.
```

Comment:
Does the 'X is the interval of sending in seconds' always means 'X means sending data every X seconds'? Can it be interpreted like 'how many seconds may be consumed to transmit the data over the network'?

The same question about the metrics collect interval.

Reply:
> Can it be interpreted like 'how many seconds may be consumed to transmit the data over the network'?

I believe it cannot. @lenkis what do you think?

Reply:
I'd never think about this kind of interpretation, really. I'd leave the phrase as is.

Reply:
No problems. Thanks for taking care :)